### PR TITLE
Revising Kubelet Workbook

### PR DESCRIPTION
**Pull request by austonli**
Adding in metrics & visualizations to include more Prometheus metrics

### CODE REVIEW

File: Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook [hunk 1]
```diff
@@ -151,152 +152,720 @@
...
...essfulOperationsSeries = series_less(TotalSuccessfulOperationsSeries, SeriesOfZeroes)\r\n| extend TotalSuccessfulOperationsSeries = array_iif(IsNegativeTotalSuccessfulOperationsSeries, SeriesOfZeroes, TotalSuccessfulOperationsSeries)\r\n| extend SuccessPercentage = round(iif(TotalOperations == 0, 1.0, iif(TotalErrors > TotalOperations, 0.0, 1 - (TotalErrors / TotalOperations))), 4), SuccessPercentageSeries = series_divide(TotalSuccessfulOperationsSeries, TotalOperationsSeries)\r\n| extend SeriesOfOneHundo = series_multiply(series_divide(SeriesOfEqualLength, SeriesOfEqualLength), 100)\r\n| extend SuccessfulOperationsEqualsTotalOperationsSeries = series_equals(TotalSuccessfulOperationsSeries, TotalOperationsSeries)\r\n| extend SuccessPercentageSeries = array_iff(SuccessfulOperationsEqualsTotalOperationsSeries, SeriesOfOneHundo, SuccessPercentageSeries)\r\n| project HostName, TotalOperations, TotalErrors, SuccessPercentage, SuccessPercentageSeries\r\n| order by SuccessPercentage asc, HostName asc\r\n| project-rename Node = HostName, ['Total Operations'] = TotalOperations, ['Total Errors'] = TotalErrors, ['Success %'] = SuccessPercentage, ['Success % Trend'] = SuccessPercentageSeries",
```
Comment (ganga1980) [inline Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook:677]: I see my recent changes to kubelet workbook not here.  can you please take latest merged changes to this work book?

File: Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook [hunk 2]
```diff
@@ -151,152 +152,720 @@
...
...xtend IsNegativeTotalSuccessfulOperationsByTypeSeries = series_less(TotalSuccessfulOperationsByTypeSeries, SeriesOfZeroes)\r\n| extend TotalSuccessfulOperationsByTypeSeries = array_iif(IsNegativeTotalSuccessfulOperationsByTypeSeries, SeriesOfZeroes, TotalSuccessfulOperationsByTypeSeries) \r\n| extend SuccessPercentage = round(iif(TotalOperations == 0, 1.0, iif(TotalErrors > TotalOperations, 0.0, 1 - (TotalErrors / TotalOperations))), 4), SuccessPercentageSeries = series_divide(TotalSuccessfulOperationsByTypeSeries, TotalOperationsByTypeSeries)\r\n| extend SuccessfulOperationsEqualsTotalOperationsSeries = series_equals(TotalSuccessfulOperationsByTypeSeries, TotalOperationsByTypeSeries)\r\n| extend SuccessPercentageSeries = array_iff(SuccessfulOperationsEqualsTotalOperationsSeries, SeriesOfOneHundo, SuccessPercentageSeries)\r\n| project OperationType, TotalOperations, TotalErrors, SuccessPercentage, SuccessPercentageSeries\r\n| order by SuccessPercentage asc, OperationType asc\r\n| project-rename ['Operation Type'] = OperationType, ['Total Operations'] = TotalOperations, ['Total Errors'] = TotalErrors, ['Success %'] = SuccessPercentage, ['Success % Trend'] = SuccessPercentageSeries",
```
Comment (ganga1980) [inline Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook:731]: Here as well . I see my recent changes to kubelet workbook not here.  can you please take latest merged changes to this work book?

**Review by ganga1980 — CHANGES_REQUESTED** [review]
auston, my recent changes to kubelet workbook not here. can you please take latest changes in repo and merge with your changes?

File: Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook (hunk 2 above)
Comment (austonli) [reply Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook]: Fixed

File: Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook [hunk 3]
```diff
@@ -6,44 +6,9 @@
       "content": {
```
Comment (nibarnet) [inline Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook:6]: Make clusterIdWhereClause required, as it needs to be in place for all queries

Comment (nibarnet) [reply Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook:6]: Make nodes adnd operations required so that the quick query parsing error doesn't show. From the error, it's clear the queries aren't running any faster because they have to wait until the parameter gets a value in order to start executing

Comment (nibarnet) [reply Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook:6]: Make the stepnames for tabs include tab in the name so it is more clear what that step is in the telemetry. E.g. "Overview" becomes "Overview Tab"

**Review by ganga1980 — APPROVED** [review]
lgtm

File: Workbooks/AKS/Node Kubelet/settings.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Kubelet",
+    "author": "Microsoft",
+    "isPreview": true,
+    "galleries": [
+        { "type": "container-insights", "resourceType": "Microsoft.ContainerService/managedClusters", "order":1100 },
+        { "type": "container-insights", "resourceType": "Microsoft.Kubernetes/connectedClusters", "order": 1100 },
```
Comment (nibarnet) [inline Workbooks/AKS/Node Kubelet/settings.json:8]: this is the wrong order number. please check the wiki. increment order # by 100 from the hightest current order number

File: Workbooks/AKS/Node Kubelet AKS Engine/Node Kubelet AKS Engine.workbook [hunk 5]
```diff
@@ -6,6 +6,13 @@
       "content": {
         "version": "KqlParameterItem/1.0",
         "parameters": [
```
Comment (nibarnet) [inline Workbooks/AKS/Node Kubelet AKS Engine/Node Kubelet AKS Engine.workbook:8]: Please revert these changes

File: Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook (hunk 3 above)
Comment (nibarnet) [reply Workbooks/AKS/Node Kubelet AKS/Node Kubelet AKS.workbook:6]: Revert changes

File: Workbooks/AKS/Node Kubelet/settings.json (hunk 4 above)
Comment (austonli) [reply Workbooks/AKS/Node Kubelet/settings.json:8]: what's the highest order # we have right now? Do I just have to click through and find it?

**Review by nibarnet — APPROVED** [review]
